### DISCUSSION TITLE
Welcome to chris.lu Discussions!

### DISCUSSION THREAD

**Discussion by chrisweb** [chrisweb/chris.lu#5] [Announcements]
## 👋 Welcome!
  I'm using Discussions as a place to connect with readers of my blog and hope that you:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others and are open-minded. Remember that this is a community we build together 🚀

**Comment by amanov-begli** [comment]
Hi Chris, since two days I am wandering through your resources and cannot stop enjoying your work. I've walked through your "Games" tab, I found so cool, with all the link, pictures. Real story, or even history of game (R)Evolution you have there. Currently reading your "little" guide on how to properly work with an AI, which definitely a perfect timing for me to discover it. Man I am just in Love with your website. Such a vibe I am feeling. Pretty much thank you for all the articles, you've written. I will read them along the way. I was happy also to discover, that you have developed your website with next.js (correct me) if I am wrong. I might have few questions about the "real" working with AI but for now, Just thank you!

**Reply by chrisweb** [reply]
Thank you for the kind words, I am so glad you like the website. Yes it's Next.js, there is no database behind it though, its fully static. The whole source code is here: https://github.com/chrisweb/chris.lu. Yeah the AI assisted coding article does well in terms of visits, I think a lot of people are wondering these days if AI is a blessing or a curse, and try to take the good parts and avoid the bad ones.

**Reply by begli-amanov** [reply]
Hi again! I am replying you now from my personal Account. Funny thing that I have a website (static too) written in next.js. I would happy hear your thoughts about it, if you don't mind. [amanov.net ](https://amanov.net/). Whatever critic or feedback you have, I'll take it. Thank you and have a great one! 

P.S.

If this is not the correct place for such conversations, I would happily move to whatever better suited place. Cheers!

**Reply by chrisweb** [reply]
I like it. It’s simple, but it does the job. It feels professional, which was probably what you were going for. I also like the quotes. They add a personal touch without deviating from the professional approach.

Blogging is difficult these days because companies look you up, and sometimes having one opinion that’s a bit too strong can cost you a great opportunity. Mine is more personal, but I also refrain from posting my thoughts too often because I don’t want to be judged for them.

From a technical point of view, it loads quickly, which is probably the most important part. It also has JSON-LD, so in terms of SEO, you’ve done everything right. The font is nicely big, which I always appreciate, as it makes everything easier to read.

And yes this is the right place to talk / post feedback. As I said my blog is static so I have no comments system, which is why I redirect people here, its not ideal, but also not bad.

**Reply by begli-amanov** [reply]
Very much appreciate your time and attention on that topic! With the website I've realized my long dream to have something I truly own. Is there anything which you don't like or would suggest to change? Don't worry, I know that most things might feel subjective, but one thought could lead to another and something good can born out of it.

So as an exchange for your time I have a feedback about your website as well! I LOVE IT for 99%. The colors, pictures/images, cursor in some gaming manner vibrant neon colors, this mind blowing infinity road in Thron style *at least for me it is like if it would be from there*, music. It is really Incredible. 

Now the .5%  from the remaining 1%. In my eyes, you should place this dashboard somewhere, where it is easier to find. I know, you have given several ways to contact you, but look, for me it was easier to write you here. It could be the same for some other people. Maybe placed it a bit more higher on the home tab, where it will be easier to find.

The next .5% is the headers navbar. Not that is not good, but just the rest of the website is SOOOO good that the header feels for me personally a bit off the case, as if it would belong to some other website. 

Nevertheless I am on your website everyday, and when I am not reading there, I hit the play button and enjoy this techno music and get pure joy!

Cheers and stay healthy!
